docs/pages/develop/user-interface/safe-areas.mdx

```diff
@@ -1,69 +1,114 @@
 
 <Terminal cmd={['$ npx expo install react-native-safe-area-context']} />
 
+</Collapsible>
+
 ### Usage
 
-#### Add `SafeAreaProvider`
+<Step label="1">
 
-To use the library, you first need to import [`<SafeAreaProvider>`](https://github.com/th3rdwave/react-native-safe-area-context#safeareaprovider) in your app's root component.
+Import and add [`SafeAreaProvider`](https://github.com/th3rdwave/react-native-safe-area-context#safeareaprovider). If your project uses Expo Router, add it to the [root layout (**app/\_layout.tsx**)](/develop/file-based-routing/#_layout-file) file. Otherwise, add it to the root component (such as **App.tsx** file).
 
-```jsx App.js
-import { View, Text } from 'react-native';
+```tsx app/_layout.tsx
 import { SafeAreaProvider } from 'react-native-safe-area-context';
 
-export default function App() {
+function RootLayout() {
   return (
     <SafeAreaProvider>
-      <View>
-        <Text>My App</Text>
-      </View>
+      <Stack>
+        <Screen name="index" options={{ headerShown: false }} />
+      </Stack>
     </SafeAreaProvider>
   );
 }
 ```
 
-Then you can use the [`<SafeAreaView>`](https://github.com/th3rdwave/react-native-safe-area-context#safeareaview), which is a regular `<View>` with the safe area insets applied as padding or margin.
+> **Note:** In the above example, the `Stack` is wrapping one screen. Your project's root layout file may have more than one screen. The `index` screen is used here an example for a screen component.
+
+</Step>
+
+<Step label="2">
+
+Use [`SafeAreaView`](https://github.com/th3rdwave/react-native-safe-area-context#safeareaview) to wrap the content of your screen's component. It is a regular `<View>` with the safe area insets applied as extra padding or margin.
+
+```tsx app/index.tsx
+import { Text } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+export default function HomeScreen() {
+  return (
+    <SafeAreaView style={{ flex: 1 }}>
+      <Text>Content is in safe area.</Text>
+    </SafeAreaView>
+  );
+}
+```
+
+</Step>
 
-#### Use `useSafeAreaInsets` hook
+## Alternate: `useSafeAreaInsets` hook
 
-Alternate to `<SafeAreaView>`, you can also use [`useSafeAreaInsets`](https://github.com/th3rdwave/react-native-safe-area-context#usesafeareainsets) hook that gives direct access to the safe area insets. It offers more flexibility and gives you more control. You can apply padding for each edge using an inset from this hook. The hook provides the insets in the following form:
+Alternate to `SafeAreaView`, you can use [`useSafeAreaInsets`](https://github.com/th3rdwave/react-native-safe-area-context#usesafeareainsets) hook in your screen component. It gives direct access to the safe area insets. It offers more flexibility and gives you more control. You can apply padding for each edge using an inset from this hook directly to the `<View>`.
```

Comment:
```suggestion
Instead of `SafeAreaView`, you can use the [`useSafeAreaInsets`](https://github.com/th3rdwave/react-native-safe-area-context#usesafeareainsets) hook in your screen component. This hook provides direct access to the safe area insets, which you can use to apply padding for each edge as an inline style on the `<View>`.
```

Reply:
The changes you're seeing here are outdated. This section was already updated. I've rebased the latest changes from the main it should reflect that now.